app/bundles/LeadBundle/Segment/Decorator/Date/Other/DateAnniversary.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\LeadBundle\Segment\Decorator\Date\Other;
+
+use Mautic\LeadBundle\Segment\ContactSegmentFilterCrate;
+use Mautic\LeadBundle\Segment\Decorator\DateDecorator;
+use Mautic\LeadBundle\Segment\Decorator\FilterDecoratorInterface;
+
+class DateAnniversary implements FilterDecoratorInterface
+{
+    /**
+     * @var DateDecorator
+     */
+    private $dateDecorator;
+
+    public function __construct(DateDecorator $dateDecorator)
+    {
+        $this->dateDecorator = $dateDecorator;
+    }
+
+    public function getField(ContactSegmentFilterCrate $contactSegmentFilterCrate)
```

Comment:
Missing docblocks for this method and those below

Reply:
Fixed